nni/retiarii/nn/pytorch/api.py

```diff
@@ -10,14 +10,13 @@
 
 from nni.common.serializer import Translatable
 from nni.retiarii.serializer import basic_unit
-from nni.retiarii.utils import NoContextError
-from .utils import generate_new_label, get_fixed_value
+from .utils import Mutable, generate_new_label, get_fixed_value
```

Comment:
why `Mutable` is put into utils?

Reply:
Because it's currently an implementation trick.

Will formalize it into an interface when necessary.